src/CsvHelper.Excel/ExcelParser.cs

```diff
@@ -104,9 +104,9 @@ public class ExcelParser : IParser
         /// <param name="stream">The stream.</param>
         /// <param name="sheetName">The sheet name</param>
         /// <param name="configuration">The configuration.</param>
-        public ExcelParser(Stream stream, string sheetName, CsvConfiguration configuration)
+        public ExcelParser(Stream stream, string sheetName, CsvConfiguration configuration, bool leaveOpen = false)
         {
-            var workbook = new XLWorkbook(stream, XLEventTracking.Disabled);
+            var workbook = new XLWorkbook(stream);
```

Comment:
In the input file is not an Excel file, but csv file. it will throw an exception
I thought the orignal idea was to support both csv and excel

Reply:
The original would have crashed there as well. 
This issue is probably better suited for a different PR. This PR merely updates the ClosedXml and CsvHelper libraries, with fixes to some breaking changes.